src/libcore/num/mod.rs

```diff
@@ -132,15 +132,11 @@ macro_rules! int_impl {
         ///
         /// Leading and trailing whitespace represent an error.
         ///
-        /// # Arguments
-        ///
-        /// * src - A string slice
-        /// * radix - The base to use. Must lie in the range [2 .. 36]
-        ///
-        /// # Return value
+        /// # Examples
         ///
-        /// `Err(ParseIntError)` if the string did not represent a valid number.
-        /// Otherwise, `Ok(n)` where `n` is the integer represented by `src`.
+        /// ```
+        /// assert_eq!(Some(10), u32::from_str_radix("A", 16));
```

Comment:
I think that the ordering is generally `assert_eq!(actual, expected)`, so perhaps this could be swapped? Other than that though r=me.